shared/fs/filepreview/index.js

```diff
@@ -1,131 +1,51 @@
-      <Box style={filePreviewHeaderStyle}>
-        <Text type="BodyBig">{props.pathItem.name}</Text>
-        {!isMobile && (
-          <PathItemInfo
-            lastModifiedTimestamp={props.pathItem.lastModifiedTimestamp}
-            lastWriter={props.pathItem.lastWriter.username}
-            startWithLastModified={true}
-          />
-        )}
-      </Box>
-      <Box style={stylesHeaderIcons}>
-        {!isMobile && <Icon type="iconfont-finder" style={stylesHeaderIcon} onClick={props.onShowInFileUI} />}
-        <Icon type="iconfont-ellipsis" style={stylesHeaderIcon} onClick={props.onAction} />
-      </Box>
-    </Box>
-    <Box style={stylesGreyContainer}>
-      <Box style={stylesContainer}>
-        <PathItemIcon spec={props.itemStyles.iconSpec} style={{}} />
-        <Text type="BodyBig" style={stylesFilename(props.itemStyles.textColor)}>
-          {props.pathItem.name}
-        </Text>
-        <Text type="BodySmall">{Constants.humanReadableFileSize(props.pathItem.size)}</Text>
-        {isMobile && (
-          <PathItemInfo
-            lastModifiedTimestamp={props.pathItem.lastModifiedTimestamp}
-            lastWriter={props.pathItem.lastWriter.username}
-            startWithLastModified={true}
-          />
-        )}
-        {isMobile && (
-          <Text type="BodySmall" style={stylesNoOpenMobile}>
-            This document can not be opened on mobile. You can still interact with it using the ••• menu.
-          </Text>
-        )}
-        {// Enable this button for desktop when we have in-app sharing.
-        isMobile && (
-          <Button
-            key="share"
-            type="Primary"
-            label="Share"
-            style={{marginTop: globalMargins.medium}}
-            onClick={props.onShare}
-          />
-        )}
-        {isIOS ? (
-          Constants.isMedia(props.pathItem.name) && (
-            <Button
-              key="open"
-              type="Secondary"
-              label={'Save'}
-              style={{marginTop: globalMargins.small}}
-              onClick={props.onSave}
-            />
-          )
-        ) : props.fileUIEnabled ? (
-          <Button
-            key="open"
-            type="Secondary"
-            label={'Show in ' + fileUIName}
-            style={{marginTop: globalMargins.small}}
-            onClick={props.onShowInFileUI}
-          />
-        ) : (
-          <Button
-            key="open"
-            type="Secondary"
-            label="Download a copy"
-            style={{marginTop: globalMargins.small}}
-            onClick={props.onDownload}
-          />
-        )}
-      </Box>
-    </Box>
-    <Footer />
-  </Box>
-)
-
-const stylesClose = {
-  marginLeft: globalMargins.tiny,
+const getDisplayComponent = (path: Types.Path, fileViewType: Types.FileViewType) => {
+  switch (fileViewType) {
+    case 'text':
+      return <TextView url="https://keybase.io/warp/release.txt" />
```

Comment:
real path right

Reply:
It's not loading real data yet. This is just a placeholder that has text.